app/concepts/api/v1/users/registrations/contract/validate.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Contract
+  class Validate < ApplicationContract
+    params do
+      required(:email).filled(:string)
+      required(:password).filled(:string)
```

Comment:
Were rule for password security size, etc.? Also good practice to have `password_confirmation` field.

Reply:
Added password confirmation and password min length